app/src/androidTest/java/ch/sdp/vibester/database/PersistanceSetterTest.kt

```diff
@@ -6,7 +6,7 @@ import org.junit.Test
 class PersistanceSetterTest {
     @Test
     fun testPersistance() {
-        PersistanceSetter.setPersistance()
-        assertEquals(true, PersistanceSetter.getPersistance())
+        PersistenceSetter.setPersistence()
+        assertEquals(true, PersistenceSetter.getPersistence())

```

Comment:
Nice catch of the spelling mistake, you can also maybe rename the file for more coherence😅